api/src/main/java/org/openmrs/module/htmlformentryui/tag/UiIncludeTagHandler.java

```diff
@@ -99,28 +99,28 @@ public void doEndTag(FormEntrySession formEntrySession, PrintWriter out, Node pa
     protected void doIncludeFragment(Node node, UiUtils uiUtils, FormEntrySession formEntrySession, PrintWriter out, String provider) {
     	String fragment = getAttribute(node, "fragment", null);
         if (StringUtils.isNotEmpty(fragment)) {
-        	Map<String, Object> props = new HashMap<String, Object>();
+        	Map<String, Object> fragmentConfig = new HashMap<String, Object>();
         	try {
         		URI fragmentUrl = new URI(fragment);
 	        	if (fragmentUrl.getQuery() != null) {
-	        		props.putAll(parseFragmentParams(fragment, formEntrySession));
+	        		fragmentConfig.putAll(paramsToMap(formEntrySession.evaluateVelocityExpression(fragment)));
 	        	}
 	        	
 	        	String fragmentParams = getAttribute(node, "fragmentParams", null);
 	        	if (StringUtils.isNotEmpty(fragmentParams)) {
 	        		StringBuilder urlBuilder = new StringBuilder();
 	        		urlBuilder.append(fragmentUrl.getPath() + "?").append(fragmentParams);
-	        		props.putAll(parseFragmentParams(urlBuilder.toString(), formEntrySession));
+	        		fragmentConfig.putAll(paramsToMap(formEntrySession.evaluateVelocityExpression(urlBuilder.toString())));
 	        	}
         	} catch(URISyntaxException e) {
         		log.error("Invalid fragment URL", e);
         	} 
         	
             try {
-				if (props.isEmpty()) {
+				if (fragmentConfig.isEmpty()) {
```

Comment:
Do we really need this if, can't `includeFragment` deal with a null-or-empty config?